airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/postgres/test_nested_streams/models/generated/airbyte_incremental/test_normalization/nested_stream_with_c___names_partition_data.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(
+    indexes = [{'columns':['_airbyte_emitted_at'],'type':'hash'}],
+    schema = "test_normalization",
+    tags = [ "nested" ]
+) }}
```

Comment:
Un-nesting an array type column from the parent stream can NOT re-use the same `_airbyte_ab_id` column as unique_key... each element of the nested array would have the same unique_key and it wouldn't be unique anymore...